plugins/catalog-graph/EXPERIMENTAL.md

```diff
@@ -0,0 +1,224 @@
+
+#### How to override the card extension
+
+There is more than one option on where to place extension overrides, see the official [documentation](https://backstage.io/docs/frontend-system/architecture/extension-overrides) to learn more about the recommendations.
+
+> [!Warning]
+> To maintain the same level of configuration, you should define the same or an extended configuration schema.
+
+```tsx
+import { createExtensionOverrides } from '@backstage/backstage-plugin-api';
+import { createEntityCardExtension } from '@backstage/plugin-catalog-react/alpha';
+
+createExtensionOverrides(
+  extensions: [
+    createEntityCardExtension({
+      // This namespace is necessary so the system knows that this extension will replace the default 'entity-relations' card extension that is provided by the 'catalog-graph' plugin
+      namespace: 'catalog-graph',
+      name: 'entity-relations',
+      loader: () => import('./components').then(m => <m.MyEntityRelationsCard />)
+    })
+  ]
+);
+```
+
+### Catalog Entity Relations Graph Page
+
+This page displays a more detailed relation graph for a Catalog entity. It contains a few filters that you can use to narrow down what's displayed.
+
+| kind | namespace     | name | id                   |
+| ---- | ------------- | ---- | -------------------- |
+| page | catalog-graph | -    | `page:catalog-graph` |
+
+#### Output
+
+A React component defined by the [coreExtensionData.reactElement](https://backstage.io/docs/reference/frontend-plugin-api.coreextensiondata/) type.
+
+#### Inputs
+
+There are no inputs available for this extension.
+
+#### Config
+
+The page configurations should be defined under the `app.extensions.page:catalog-graph.config` key:
+
+```yaml
+# example configuring the page path
+app:
+  extensions:
+    # this is the extension id and it follows the naming pattern bellow:
+    # <extension-kind>/<plugin-namespace>:<extension-name>
+    - page:catalog-graph:
+        config:
+          # defaults to "/catalog-graph"
+          path: '/entity-graph'
+```
+
+See below the complete list of available configs:
+
+| Key             | Description                                                                                                                | Type                                                                                                                                                                                                                                                | Optional | Default value                                                                                                               |
```

Comment:
I would appreciate someone more experienced with the Catalog plugin to help me write better descriptions for these configs.